orbyter-spark-dev/VERSION

```diff
@@ -0,0 +1 @@
+0.1
```

Comment:
I think we are trying to keep version numbers for similar install packages constant across the `ml-dev`, `dl-dev`, and `spark-dev`. However, this hasn't been enforced. Mainly just commenting here that this maybe should be `3.2`. I don't know if it makes sense to enforce this when the enforcement is anecdotal.

Reply:
It also may make sense to have this version match the spark version.

Reply:
What if we make a small spark-specific change, but still use the same base package setup as `ml-dev:3.4`? I guess we could use minor versions, but that goes against the semantic versioning standard. The same issue applies with tying it to the spark version number.